app/scripts/views/confirm.js

```diff
@@ -45,6 +45,7 @@ define(function (require, exports, module) {
       var email = this.getAccount().get('email');
       var isSignIn = this.isSignIn();
       var isSignUp = this.isSignUp();
+      var openWebmailButtonVisible = this.isOpenWebmailButtonVisible(email);
```

Comment:
do you need this local variable here or can you just call `this.isOpenWebmail...` in `return {` below?